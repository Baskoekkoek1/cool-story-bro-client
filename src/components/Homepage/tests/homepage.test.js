import React from "react";
import renderer from "react-test-renderer";

import Homepage from "../index.js";

describe("<Homepage >", () => {
  test("should render without props", () => {
    // Should it? this is a decision you should make over your components.
    // Writing tests helps us think of all the open possibilities we left
    // in our code and how they would affect our app.
    const component = renderer.create(<Homepage />); // "render" the component
    const tree = component.toJSON(); // Convert it to JSON
    expect(tree).toMatchSnapshot(); // Compare to stored snapshot
  });
});
