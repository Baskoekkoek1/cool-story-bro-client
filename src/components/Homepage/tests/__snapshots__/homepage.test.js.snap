// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Homepage > should render without props 1`] = `
<div
  className="jumbotron"
  style={
    Object {
      "backgroundColor": undefined,
      "color": undefined,
    }
  }
>
  <h1 />
  <p />
</div>
`;
